samples/system-test/detect.v2beta1_test.js

```diff
@@ -0,0 +1,164 @@
+  async t => {
+    const output = await runAsync(
+      `${cmd} getKnowledgeBase -n "${knowbaseFullName}"`
+    );
+    console.log(`getKnowledgeBase: input is ${knowbaseFullName}`);
+    const parsedOut = output.split(`\n`);
+    console.log(`getKnowledgeBase: output is ${output}`);
+    t.true(
+      parsedOut[0].includes(knowbaseFullName) &&
+        parsedOut[1].includes(knowbaseDisplayName)
+    );
+  }
+);
+
+test.serial(
+  `It should locate knowledge base with listKnowledgeBases`,
+  async t => {
+    const output = await runAsync(`${cmd} listKnowledgeBases`);
+    const parsedOut = output.split(`\n`);
+    parsedOut.forEach(p => {
+      if (p.includes(testKnowledgeBaseName)) t.pass();
+    });
+    console.log(output);
+  }
+);
+
+test.serial(`It should create a document in Knowledge Base`, async t => {
+  const output = await runAsync(
+    `${cmd} createDocument -n "${knowbaseFullName}" -z "${testDocumentPath}" -m "${testDocName}"`
+  );
+  t.true(output.includes(`Document created`));
+  console.log(`Document created successfully`);
+});
+
+test.serial(
+  `It should list document just created with listDocuments`,
+  async t => {
+    const output = await runAsync(
+      `${cmd} listDocuments -n "${knowbaseFullName}"`
+    );
+    const parsedOut = output.split(`\n`);
+    documentFullPath = parsedOut[parsedOut.length - 1].split(`:`)[1];
+    t.true(documentFullPath !== ``);
+    console.log(`Confirming documentID: ${documentFullPath}`);
+  }
+);
+
+test.serial(`It should detect Intent with Text to Speech Response`, async t => {
+  const output = await runAsync(
+    `${cmd} detectIntentwithTexttoSpeechResponse -q "${testQuery}"`
+  );
+  const parsedOut = output.split(`\n`);
+  t.true(
+    parsedOut[1].includes(
+      `Audio content written to file: ./resources/output.wav`
+    )
+  );
+  console.log(`confirm audio file generated: ${parsedOut[1]}`);
+});
+
+test.serial(`It should detect Intent Knowledge`, async t => {
+  const output = await runAsync(
+    `${cmd} detectIntentKnowledge -q "${testQuery}"`
+  );
+  const parsedOut = output.split(`\n`);
+  // documentFullPath = parsedOut[parsedOut.length - 1].split(`:`)[1];
+  t.true(parsedOut[parsedOut.length - 3] !== null);
+  console.log(`confirm query result: ${parsedOut[parsedOut.length - 5]}`);
+});
+
+test.serial(`It should detect sentiment with intent`, async t => {
+  const output = await runAsync(
+    `${cmd} detectIntentandSentiment -q "${testQuery}"`
+  );
+  const parsedOut = output.split(`\n`);
+  t.true(parsedOut[parsedOut.length - 3].includes(`Detected sentiment`));
+  console.log(`confirm sentiment result: ${parsedOut[parsedOut.length - 2]}`);
+  console.log(`confirm sentiment result: ${parsedOut[parsedOut.length - 1]}`);
+});
+
+test.serial(`It should detect Intent with Model Selection`, async t => {
+  const output = await runAsync(`${cmd} detectIntentwithModelSelection`);
+  const parsedOut = output.split(`\n`);
+  t.true(parsedOut[1] !== null);
```

Comment:
This is not an acceptable assertion. Please assert that the output includes something valuable, not that it is simply not null